ui/tract-renderer/src/main/java/org/cru/godtools/tract/activity/TractActivity.kt

```diff
@@ -62,12 +64,7 @@ import org.cru.godtools.tract.util.ViewUtils
 import org.cru.godtools.tract.util.isTractDeepLink
 import org.cru.godtools.tutorial.PageSet
 import org.cru.godtools.tutorial.activity.buildTutorialActivityIntent
-import org.cru.godtools.xml.model.Card
-import org.cru.godtools.xml.model.Modal
-import org.cru.godtools.xml.model.Page
-import org.cru.godtools.xml.model.backgroundColor
-import org.cru.godtools.xml.model.navBarColor
-import org.cru.godtools.xml.model.navBarControlColor
+import org.cru.godtools.xml.model.*
```

Comment:
the wildcard import will trigger a ktlint violation, you'll have to adjust your Android Studio code style settings to prevent it from automatically happening